weblab/templates/fitting/fittingresultversion_detail.html

```diff
@@ -0,0 +1,111 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load experiments %}
+
+{% block title %}Fitting result details - {% endblock title %}
+
+{% block body_id %}experiment-version{% endblock %}
+
+{% block content %}
+  {% with version.fittingresult as fittingresult %}
+  {% include "./includes/fittingresult_header.html" %}
+
+  <div id="entitydetails"></div>
+
+  <div id="entityversion"
+       data-version-href="{% url 'fitting:result:version' fittingresult.id version.id %}"
+       data-version-json-href="{% url 'fitting:result:version_json' fittingresult.id version.id %}">
+    <div class="closebtn">
+      <small>
+        <a href="{% url 'fitting:result:versions' fittingresult.id %}">list all versions</a>
+      </small>
+    </div>
+    <h2 id="entityversionname">
+      <small>Version:</small> {{ version.name }}
+    </h2>
+    <div class="suppl">
+      <small>
+        Created <time id="entityversiontime">{{ version.created_at }}</time>
+        by <em id="entityversionauthor">{{ version.author.full_name }}</em>.
+        {% if version.finished_at %}Took {{ version.created_at|timesince:version.finished_at }}.{% endif %}
+        Visibility: <span title="This is derived from the corresponding model version and protocol version visibilities">{{ version.visibility }}</span>
+        <img src="{% static 'img/info.png' %}" alt="help" title="{{ VISIBILITY_HELP }}" />.
+        <span id="exptStatus"></span>
+
+        {% if perms.run_fits %}
+          <a id="rerunExperiment"><img src="{% static 'img/refresh.png' %}" alt="rerun experiment" title="rerun experiment"/></a> <span id="rerunExperimentAction"></span>
```

Comment:
The JS action associated with this doesn't work - it tries to do a normal experiment run, not a fitting experiment.